week1/week1-assigment/index.js

```diff
@@ -0,0 +1,52 @@
+// To run this file: node index.js
+// Then in your browser: http://localhost:8080
+var http = require('http');
+
+var port = 8080;
+let status = 10;
+var server = http.createServer();
+
+// Start the HTTP server, start listening for requests
+server.listen(port, function (error) {
+    if (error) {
+        console.log(error);
+    } else {
+        console.log('api listening on port', port);
+    }
+});
+
+// Create a event handler for "request"
+// this is an alternative way
+server.on('request', function (request, response) {
+
+
+    if (request.url === "/state") {
```

Comment:
`if/elseif/else` works...okay...for this, but can you think of a better way to handle all the different cases?

Reply:
switch you mean is better for cases right ?
i already check it out bt there is problem and am still tryna figure out it

Reply:
Yes, `switch` is what I have in mind. Good luck 😄